### PR TITLE
Config with default template

### PR DESCRIPTION
**Pull request by gforcada**
@mauritsvanrees I replaced the existing `tox.ini` and its related files that were used to run GHAs. As it is only testing on Plone 6.0.x, that's the same we do with `plone/meta`.

We are losing the python matrix, but we can set that up as an environment variable.

Let me try it 💥

### CODE REVIEW

**Comment by gforcada** [comment]
@mauritsvanrees see https://github.com/plone/meta/pull/76 once it is merged, the `TEST_PYTHON_VERSIONS` is already customized for this repository on https://github.com/plone/Products.isurlinportal/settings/variables/actions to restore the 3.8-3.11 range 😄

File: setup.py [hunk 1]
```diff
@@ -36,5 +36,10 @@
     include_package_data=True,
     zip_safe=False,
     python_requires=">=3.8",
-    install_requires=["setuptools"],
+    install_requires=[
+        "Products.CMFPlone",
+        "setuptools",
+        "plone.base",
+        "plone.registry",
```
Comment (jensens) [inline setup.py:43]: not needed, see plone/meta#79

File: setup.py [hunk 2]
```diff
@@ -36,5 +36,10 @@
     include_package_data=True,
     zip_safe=False,
     python_requires=">=3.8",
-    install_requires=["setuptools"],
+    install_requires=[
+        "Products.CMFPlone",
```
[the remark this answers is not in the document]
Comment (gforcada) [reply setup.py:40]: Then, we can leave it out, and only require it on testing? 🤔 or fix the circular dependency 😅

[the remark this answers is not in the document]
Comment (jensens) [reply setup.py:40]: @mauritsvanrees as you introduced the package  initially (IIRC), do you have a good idea how to solve this?

**Comment by mauritsvanrees** [comment]
> @mauritsvanrees I replaced the existing `tox.ini` and its related files that were used to run GHAs. As it is only testing on Plone 6.0.x, that's the same we do with `plone/meta`.
> 
> We are losing the python matrix, but we can set that up as an environment variable.
> 
> Let me try it 💥

That seems good to me.  I don't see it working yet in this PR, but that may be work in progress. :-)

**Comment by gforcada** [comment]
Cool, indeed the python matrix GHA work 🎉

File: setup.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (mauritsvanrees) [reply setup.py]: I have pushed a commit to move CMFPlone to a new `test` extra.

My comment for further cleanup in Plone 6.1 is moved to issue #12.

**Review by mauritsvanrees — APPROVED** [review]

**Comment by mauritsvanrees** [comment]
@jenkins-plone-org please run jobs

**Comment by gforcada** [comment]
Thanks for the fixes, as you are in the process of cutting a release I was holding back to merge more changes 😅

**Pull request merged**